include/RAJA/RAJA-config.cmake.in

```diff
@@ -0,0 +1,61 @@
+# - RAJA: (David's TO DO)
```

Comment:
All blame for this can be laid at the feet of David Boehme, I stole it perfectly, he just wrote it wrong :)